router/worker.go

```diff
@@ -806,7 +821,7 @@ func (w *worker) sendRouterResponseCountStat(status *jobsdb.JobStatusT, destinat
 	destinationTag := misc.GetTagName(destination.ID, destination.Name)
 	var alert bool
 	alert = w.allowRouterAbortedAlert(errorAt)
-	if status.JobState == jobsdb.Succeeded.State {
+	if status.JobState == jobsdb.Succeeded.State || status.JobState == jobsdb.Filtered.State {
```

Comment:
Any adaptations needed in `sendEventDeliveryStat`?

Reply:
IMO, No.

router response counts -> responses router received for an event. respStatusCode label will reveal about the status of the delivery.

event delivery stats -> stats for successfully delivered events (count & time spent by the event in the pipeline since it is received.)